src/app/backend/deploy.go

```diff
@@ -224,3 +237,28 @@ func getLabelsMap(labels []Label) map[string]string {
 
 	return result
 }
+
+// Deploys an app based on the given yaml or json file.
+func DeployAppFromFile(spec *AppDeploymentFromFileSpec, client *client.Client) error {
+	// Creates temp file to use it in kubectl command
+	file, err := ioutil.TempFile("", spec.Name)
```

Comment:
I'm very worried about creating the temporary file. I dont even know if it is supported in our docker scratch image. 

How about we do copy->paste->remove_unneeded_stuff from the RunCreate method? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create.go#L90

I think that we can have a RunCreate method that reads from a stream in a dozen of lines, because most of the stuff there is not needed for us. 

The data would be read from a stream: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/resource/builder.go#L150
Instead from a file. Basically the part of RunCreate needs to change:
```go
	r := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
		Schema(schema).
		ContinueOnError().
		NamespaceParam(cmdNamespace).DefaultNamespace().
		FilenameParam(enforceNamespace, options.Filenames...).
		Flatten().
		Do()
```

What do you think?